src/web/app/pom.xml

```diff
@@ -735,7 +735,7 @@
       <id>teradata</id>
```

Comment:
Should this profile just be removed?

Reply:
Should be fine to keep it, like the other profiles we have for community modules. It allows a developer in the IDE, or anyone building GeoServer from Maven, to include the module in WAR.
Will be removed once the community module is also removed (given the trajectory of this module, probably September 2022?)